modules/29_CES_parameters/calibrate/datainput.gms

```diff
@@ -241,50 +235,39 @@ $include "./modules/29_CES_parameters/calibrate/input/pm_fe_demand_EDGETbased.cs
 $offdelim
 /
 ;
-
-
 $ifthen "%cm_calibration_FE%" == "low"
-  pm_fedemand(t,regi,"%cm_GDPscen%",in_dyn35)$(t.val ge 2006) = p29_fedemand_trasp(t,regi,"gdp_SSP2","MixWise",in_dyn35);
+  pm_fedemand(t,regi,in_dyn35)$(t.val ge 2006) = p29_fedemand_trasp(t,regi,"gdp_SDP","ElecEraWise",in_dyn35);
```

Comment:
It is unfortunate that EDGE-T couples the SSP and the demand-level dimension. A SSP2 low-demand variant might differ from an SDP low-demand one... :( But for the moment I do not see any way around this.